app/components/PercentageStats/index.tsx

```diff
@@ -39,13 +37,25 @@ function PercentageStats(props: PercentageStatsProps) {
             headingSize={headingSize}
             headerDescription={headerDescription}
         >
-            <CompactInformationCard
-                className={styles.statCard}
-                icon={StatValueIcon}
-                label={StatValueDescription}
-                spacing={StatValueSpacing}
-                value={StatValue}
-            />
+            <div className={styles.valueDescription}>
+                <div className={styles.iconContainer}>
+                    {icon}
+                </div>
+                <div className={styles.valueAndSubValue}>
+                    <NumberOutput
+                        className={styles.valueText}
+                        value={statValue}
+                        suffix={suffix === 'percentage' ? '%' : ''}
```

Comment:
```suggestion
                        suffix={suffix}
```

Let's just accept a string as suffix if there's more than one suffix. Or else, let;s name the prop 'suffixType' instead of 'suffix'.

Reply:
Done